trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -195,7 +197,12 @@ Implementation should follow the following rules when parsing headers with an un
 
 # Tracestate field
 
-The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+The `tracestate` HTTP header field conveys information about request
+position in multiple distributed tracing graphs. This header is a
+companion header for the `traceparent`. If library or platform failed to
+parse `traceparent` - it MUST NOT attempt to parse the `tracestate`.
+Note, that opposite it not true - failure to parse `tracestate` MUST NOT
```

Comment:
Would removing "Note, that opposite is not true -" make the doc easier to read?

Reply:
What if we break it up into multiple paragraphs, mentioning that `tracestate` adds information to the trace we create using the `traceparent`? Emphasising that `tracestate` is additional optional information, but `traceparent` is required, if there is a `tracestate`.

Reply:
calling `tracestate` optional can be misleading as it may be read as "optional to propagate". This paragraph is about parsing and validation rules.